### PR TITLE
Also install .cmt .cmti .ml files.

### PR DESCRIPTION
**Pull request by vbmithr**

### CODE REVIEW

**Comment by vbmithr** [comment]
What about this?

File: Makefile.in [hunk 1]
```diff
@@ -20,7 +20,12 @@ EXT_LIB = @EXT_LIB@
 EXT_OBJ = @EXT_OBJ@
 EXT_DLL = @EXT_DLL@
 
+BIN_ANNOT = $(shell expr `echo $OCAMLVERSION | cut -f1 -d.` \>= 4)
```
Comment (ygrek) [inline Makefile.in]: I don't see how this can possibly work. This test is always true.
Please use configure and define OCAML_HAS_BIN_ANNOT feature there.

**Comment by vbmithr** [comment]
What about this?

**Comment by ygrek** [comment]
Thanks, committed as ab558de

**Pull request closed without merging**